ext/search_kit/css/crmSearchDisplayTable.css

```
@@ -31,6 +31,7 @@
 table.crm-sticky-header > thead > tr {
   position: sticky !important;
   top: var(--crm-menubar-bottom, 0px);
+  z-index: 1;
```

Wouldn't this need to be > 1, since the editable elements already have a z-index of 1?
```suggestion
  z-index: 2;
```

---

I think any z-index > 1 should work because it creates a different stacking context from the table contents, and because the table contents effectively have a z-index = auto = 0.

---

There are a few elements in the table body with z-index=1, including one being added in #31408 so I think 2 might be a safer bet.